bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/page/chat-message/message-toolbar/component.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import Popover from '@mui/material/Popover';
+import { layoutSelect } from '/imports/ui/components/layout/context';
+import Button from '/imports/ui/components/common/button/component';
+import BBBMenu from '/imports/ui/components/common/menu/component';
+import { Layout } from '/imports/ui/components/layout/layoutTypes';
+import { ChatEvents } from '/imports/ui/core/enums/chat';
+import { defineMessages, useIntl } from 'react-intl';
+import {
+  Container,
+  EmojiPicker,
+  EmojiPickerWrapper,
+  EmojiButton,
+} from './styles';
+
+const intlMessages = defineMessages({
+  reply: {
+    id: 'app.chat.toolbar.reply',
+    description: 'reply label',
+  },
+});
+
+interface ChatMessageToolbarProps {
+  messageId: string;
+  chatId: string;
+  username: string;
+  message: string;
+  messageSequence: number;
+  onEmojiSelected(emoji: { id: string; native: string }): void;
+}
+
+const ChatMessageToolbar: React.FC<ChatMessageToolbarProps> = (props) => {
+  const {
+    messageId, chatId, message, username, onEmojiSelected, messageSequence,
+  } = props;
+  const [reactionsAnchor, setReactionsAnchor] = React.useState<Element | null>(
+    null,
+  );
+  const intl = useIntl();
+  const isRTL = layoutSelect((i: Layout) => i.isRTL);
+  const CHAT_TOOLBAR_CONFIG = window.meetingClientSettings.public.chat.toolbar;
+  const CHAT_REPLIES_ENABLED = CHAT_TOOLBAR_CONFIG.includes('replies');
```

Comment:
```suggestion
  const CHAT_REPLIES_ENABLED = CHAT_TOOLBAR_CONFIG.includes('reply');
```

I suggest naming it `reply` instead of `replies` to maintain consistency with `edit` and `delete`.